src/main/java/Baymax.java

```diff
@@ -0,0 +1,163 @@
+import java.util.ArrayList;
+import java.util.PrimitiveIterator;
+import java.util.Scanner;
+import task.*;
+public class Baymax {
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    public static void main(String[] args) {
+        //instantiate Baymax
+        Baymax baymax = new Baymax();
+
+        //Scanner object for user input
+        Scanner scanner = new Scanner(System.in);
+        String userInput;
+
+        //load Tasks from file
+        taskList = Storage.loadTasks();
+
+        //Display greeting
+        String line = "______________________________________________________________\n";
+        System.out.println(line
+                + "Hello! I'm Baymax, your personal emotional companion :)\n"
+                + "How may I help you today?\n"
+                + line);
+
+        //Input loop
+        while (true) {
+            userInput = scanner.nextLine();
+
+            //Check for end of conversation
+            if (userInput.equalsIgnoreCase("bye")) {
+                //Display exit message
+                System.out.println(line
+                        + "Goodbye. Have a nice day, and remember;\n"
+                        + "don't forget to smell the flowers, as you aim for the sky\n"
+                        + line);
+                break;
+            }
+            //Check for list
+            if (userInput.equalsIgnoreCase("list")) {
```

Comment:
Perhaps you could consider using else if statements? This would remove the need for break and continue statements.